src/css/vendor/normalize.css

```diff
@@ -11,6 +11,7 @@
  html {
     line-height: 1.15; /* 1 */
     -webkit-text-size-adjust: 100%; /* 2 */
+    scroll-behavior: smooth;
```

Comment:
As this is a vendored file, we can't make direct edits to it in this way. If this is a vital change to this PR it would need to occur elsewhere.